Contest_API.md

```diff
@@ -58,22 +58,22 @@ requested, e.g. at the URL path
 
 ` GET https://example.com/api/contests/wf14/teams`
 
-while an element with specific ID is requested as
+while an object with a specific ID is requested as
 
 ` GET https://example.com/api/contests/wf14/teams/10`
 
-A collection is always returned as a JSON list of objects. Every object
-in the list represents a single element (and always includes the ID).
-When requesting a single element the exact same object is returned. E.g.
+A collection is always returned as an array of JSON objects. Every item
+in the array is a single contest object (and always includes the ID).
+When requesting a single object the exact same object is returned. E.g.
 the URL path
 
 `GET baseurl/collection`
 
 returns
 
 ```json
-[ { "id":<id1>, <element specific data for id1> },
-  { "id":<id2>, <element specific data for id2> },
+[ { "id":<id1>, <object specific data for id1> },
+  { "id":<id2>, <object specific data for id2> },
```

Comment:
I find "object specific" a bit confusing here. How about:

```
[ { "id":<id1>, <properties for object with id=id1> },
  { "id":<id2>, <properties for object with id=id2> },
```
?

Reply:
Based on slack with Fredrik I've changed these to `{ "id":<id1>, <other properties for object id1> }`.